dungeon.js

```diff
@@ -553,7 +578,27 @@ function createPalette() {
         ul.appendChild(li);
     }
     cellMarkDiv.appendChild(ul);
+    
+    // Label Fill Colors
+    var labelFillDiv = document.getElementById("label_fill_colors");
+    labelFillDiv.innerHTML = "";
+        for (const [key, value] of Object.entries(labelFillTypes)) {
+        var btn = document.createElement("button");
+        btn.setAttribute("onclick", "changeSelectedLabelFill('" + key + "')");
+        //btn.innerHTML = key;
+        btn.name = key;
+        if (isDarkColor(value)) {
+            btn.style = "background-color: " + value + "; color:white";
+        } else {
+            btn.style = "background-color: " + value + "; color:black";
+        }
```

Comment:
I don't think the isDarkColor is actually needed here? since there isn't any text inside the button (which is the only reason for the `color:white/black` attribute)